docs/install/tutorials/rpi.rst

```diff
@@ -426,18 +433,21 @@ After replicating your SD card and external storage device, you need re-register
   # This will ask you questions
   kolibri manage provisiondevice
 
-  
-Future steps
-------------
 
-Kolibri is under development with regards to optimizing performance on Raspberry Pi. We are adding support for multiple CPU cores, and since the Raspberry Pi has 4 of these, it will benefit greatly. Tests indicate almost a factor 4.
+How many clients are supported?
+-------------------------------
 
-This work will be released in a future package targeting a pre-configured UWSGI and Nginx implementation, which will be possible to add by replacing the package of this tutorial with the new package. We will add a release note and update this tutorial accordingly.
+.. tip:: We recommend that you do your own benchmarking and share experiences in our `Community Forums <https://community.learningequality.org/>`_. See especially this thread about `choosing a MicroSD card <https://community.learningequality.org/t/microsd-cards-picking-the-right-one-experiences-and-benchmarks/935>`_.
 
-You may also want to install other services such as `Kiwix <https://www.kiwix.org>`_. If you have followed this tutorial, you can install Kiwix alongside Kolibri by downloading the ``kiwix-serve`` package and adding an Nginx configuration similar to the one we added for Kolibri.
+One of the bigger questions that may be hard to answer is how many students that can use Kolibri on a Raspberry Pi at the same time. One of the biggest bottle necks will be the Raspberry Pi's built-in Wi-Fi. Maybe it will support 5-10 devices that are located in a close perimeter.
```

Comment:
It can hard to predict how many students will be able to use Kolibri on a Raspberry Pi at the same time. A major bottleneck will be the Raspberry Pi's built-in Wi-Fi, which might support 5-10 devices located nearby.

Reply:
Thanks, and yes sloppy formulation from my side :blush: . Would you mind adding it as a suggestion?

Reply:
you're talking about the github function? I've seen people use it but don't actually see the option to use it myself.

For this PR i'll let you just copy/paste, and I'll learn how to use it for the next one if that's cool

Reply:
:+1: